docs/topics/native/native-get-started.md

```diff
@@ -155,13 +180,205 @@ performs an incremental build of the project.
    }
    ```
 
-4. Run the application.
-5. Enter your name and enjoy the result:
+3. Run the application.
+4. Enter your name and see the result:
 
    ![Application output](native-output-gutter-3.png){width=422}
 
+## Using Gradle
+
+In this section, you'll learn how to manually create a Kotlin/Native application using Gradle.
+
+### Create project files
+
+1. To get started, install the compatible version of [Gradle](https://gradle.org/install/). See the [compatibility table](gradle-configure-project.md#apply-the-plugin)
+   to check the Kotlin Gradle plugin (KGP) compatibility with available Gradle versions.
+2. Create an empty project directory. Inside it, create the `build.gradle(.kts)` Gradle build file with the following
+   content:
+
+   <tabs group="build-script">
+   <tab title="Kotlin" group-key="kotlin">
+
+   ```kotlin
+   // build.gradle.kts
+   plugins {
+       kotlin("multiplatform") version "%kotlinVersion%"
+   }
+
+   repositories {
+       mavenCentral()
+   }
+
+   kotlin {
+       macosArm64("native") {  // on macOS
+       // linuxArm64("native") // on Linux
+       // mingwX64("native")   // on Windows
+           binaries {
+               executable()
+           }
+       }
+   }
+
+   tasks.withType<Wrapper> {
+       gradleVersion = "%gradleVersion%"
+       distributionType = Wrapper.DistributionType.BIN
+   }
+   ```
+
+   </tab>
+   <tab title="Groovy" group-key="groovy">
+
+   ```groovy
+   // build.gradle
+   plugins {
+       id 'org.jetbrains.kotlin.multiplatform' version '%kotlinVersion%'
+   }
+
+   repositories {
+       mavenCentral()
+   }
+
+   kotlin {
+       macosArm64('native') {  // on macOS
+       // linuxArm64('native') // on Linux
+       // mingwX64('native')   // on Windows
+           binaries {
+               executable()
+           }
+       }
+   }
+
+   wrapper {
+       gradleVersion = '%gradleVersion%'
+       distributionType = 'BIN'
+   }
+   ```
+
+   </tab>
+   </tabs>
+
+   You can use different [target presets](native-target-support.md), such as `macosArm64`, `iosArm64` `linuxArm64`,
```

Comment:
Should we remove any mention of "presets" since we removed the "Set up targets for Kotlin Multiplatform" page?

Reply:
Oh yeah, that's a good idea, rewrote the whole paragraph